tests/integration_tests/test_server.py

```diff
@@ -53,7 +53,7 @@
     assert msg == {
         'command': 'notice',
         'style': 'error',
-        'text': 'You are banned from FAF for 981 years.\n Reason :\n Test permanent ban'}
+        'text': 'You are banned from FAF for 980 years.\n Reason :\n Test permanent ban'}
```

Comment:
Yea this is annoying. Mazor fixed it in his PR so it will say "forever" once the time reaches a certain amount.

Reply:
So I will leave it as is until that is pushed.

Reply:
So I will leave it as above to make the tests pass and adapt if their request is pulled.